include/deal.II/multigrid/multigrid.h

```diff
@@ -224,8 +278,55 @@ class Multigrid : public Subscriptor
    */
   void set_debug (const unsigned int);
 
+  /**
+   * Connect a function to Signals::transfer_to_mg.
+   */
+  boost::signals2::connection
+  connect_transfer_to_mg(const std::function<void (bool)> &slot);
```

Comment:
class `Triangulation` simply exposes the `Triangulation::signals` function so that callers can directly connect to the signals, rather than having to have a separate function like this one here. Any opinions on the matter?

Reply:
An alternative would be to split the signals for the respective `MG` classes.

Reply:
Your choice.

Reply:
In other words, this is something that I'm happy to leave to the person who puts in the work to implement it.